notebooks/integrations/llama3/rag-elastic-llama3.ipynb

```diff
@@ -0,0 +1,317 @@
+{
```

Comment:
_Commented on notebook `notebooks/integrations/llama3/rag-elastic-llama3.ipynb` Cell 1 Line 1_
```markdown
# RAG with Elastic and Llama3
```
this examples are either LangChain or Llama-index. I would update it like so and be obvious that its using these frameworks behind the scenes. 

_Respond and view the context [here](https://gitnotebooks.com/elastic/elasticsearch-labs/pull/266)._

Reply:
Updated it.